packages/patterns/index.js

```diff
@@ -71,9 +71,5 @@ export {
   kindOf,
 } from './src/patterns/patternMatchers.js';
 
-// ////////////////// Temporary, until these find their proper home ////////////
-
-export { listDifference, objectMap } from './src/utils.js';
```

Comment:
Reexport these as deprecated, with a suggestion to import directly from @endo/utils. That would preserve compat for this PR.

Reply:
Done